dansk.py

```diff
@@ -130,9 +131,13 @@ def _is_for_token(t1, t2, t3):
     yield tokens[-1]
 
 
-def decode(byteslike, errors="strict"):
+# pass True for ignore_first_line if the byteslike still contains the dansk
+# encoding comment
+def decode(byteslike, errors='replace', *, ignore_first_line):
     read_code = io.BytesIO(bytes(byteslike)).readline
-    read_code()  # discard first line, its the encoding comment
+    if ignore_first_line:
+        read_code()
```

Comment:
can we keep the `  # its the encoding comment`, so the reader know why we might want to remove a line of code?

Reply:
i think my new comment explains it, but let me add that